src/main/java/seedu/mtracker/ui/TextUi.java

```diff
@@ -59,6 +59,10 @@ public static void displayAddExitPriceInstruction() {
         System.out.println("Exit price: ");
     }
 
+    public static void displayAddPastReturnsInstruction() {
+        System.out.println("Past Returns: ");
+    }
+

```

Comment:
```suggestion
    public static void displayAddPastReturnsInstruction() {
        System.out.println("Past Returns (optional): ");
    }


```

Add the optional if this attribute is not compulsory